### ISSUE TITLE
Disable automatic escaping for landing page content strings

### ISSUE THREAD

**Issue by dotproto** [GoogleChrome/developer.chrome.com#4116]
**Is your feature request related to a problem? Please describe.**

At the moment it's not possible to style the content inside landing page macros. This is causing issues because it prevents content authors from using HTML entities (`&mdash;`) or basic Markdown styling (`*emphasis*`) inside these strings.

**Describe the solution you'd like**

It looks like the most direct solution is to update how strings inserted to process them as Markdown and avoid escaping HTML.

**site/_includes/macros/landing-deco.njk** starting on line 14
```diff
          <span>
-            {{ title }}
+            {{ title | md | safe }}
          </span>
        </h2>
        <p class="gap-top-300 gap-bottom-600 md-only">
-          {{ desc }}
+          {{ desc | md | safe }} 
        </p>
```

This change would also need to be applied to other landing macros that output similar strings.

**Describe alternatives you've considered**

The inability to use HTML entities is my primary concern. If the team would rather not support Markdown in these strings, we could just use the `safe` pipe instead of both `md` and `safe`.

The only other alternative I briefly considered is to use tags instead of macros, but I wrote this off as it would essentially be a complete rewrite of the landing page system.

**Additional context**

N/A

**Comment by devnook** [comment]
If I understand correctly, @dotproto you'd like to be able to use html and markdown in landing pages shortcodes (passing them inside string arguments), right?

**Comment by dotproto** [comment]
Correct. This issue originally presented itself when @jpmedley updated a line on the CWS program policies page. I don't remember the exact change, but it was something like this:

```diff
{{ landingSection(
  'Title',
-  '… landing pages, etc – unless otherwise noted. …',
+  '… landing pages, etc&mdash;unless otherwise noted. …',
  …
}} 
```

The ampersand in that HTML entity was automatically escaped when the page rendered, so the `&mdash;` appeared in the text of the rendered page. That was the primary motivation for this issue.

Markdown rendering is a secondary concern, but seemed like a nice to have. While I was working on this page, I wanted to experiment with using `_emphasis_` and `**strong**` to decorate the description string. This wasn't a hard requirement, though, so I moved on. I mostly included it because I came across the pattern of using `var | md | safe` rather than just `var | safe` while investigating the HTML entity rendering issue.
